### ISSUE TITLE
Poor contrast on pretalx pages

### ISSUE THREAD

**Issue by MarkNahabedian** [pretalx/pretalx#1479]
### Problem and impact

This is an accessibility issue.

I just sent this complaint to the JuliaCon organizers:

I have poor eyesight.  I'm struggling to read https://pretalx.com/juliacon2023/featured/ because the text is light gray and low density and the page background is white.

Because that page is presented using thus package, they referred me here.

### Expected behaviour

I expect web pages to have reasonable contrast so that they can be read by those with less than perfect eyesight.

### Steps to reproduce

1.
2.
3.
4.

### Browser (software, desktop or mobile?) and version

Android Chrome

**Comment by rixx** [comment]
Interesting – could you add a screenshot of this? This is what it looks like to me:



While the text colour is not quite black, it's not (meant to be) light gray, either. The colour comes out to #212121, which seems to be [called](https://www.color-name.com/hex/212121) "raisin black". This colour was chosen because it has a [colour contrast](https://coolors.co/contrast-checker/212121-ffffff) of 16.10 – for reference, for small text, a contrast of 7 is meant to be sufficient for small text, according to AAA level of WCAG.

For reference, organisers can override the colour, for example by uploading a custom CSS containing the line `body { color: black !important }`.

**Comment by MarkNahabedian** [comment]
I would think that for smaller text the contrast would need to be greater than for larger text.

![Screenshot_20230508-183520](https://user-images.githubusercontent.com/33787283/236952888-97e509ab-926c-4819-83ce-5978c081e839.png)

**Comment by rixx** [comment]
Yes, the colour contrast I cited is from the [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) and refers to smaller fonts:

> A contrast ratio of 3:1 is the minimum level recommended by [[ISO-9241-3]] and [[ANSI-HFES-100-1988]] for standard text and vision. The 4.5:1 ratio is used in this provision to account for the loss in contrast that results from moderately low visual acuity, congenital or acquired color deficiencies, or the loss of contrast sensitivity that typically accompanies aging. 

[…]

> The contrast ratio of 7:1 was chosen for level AAA because it compensated for the loss in contrast sensitivity usually experienced by users with vision loss equivalent to approximately 20/80 vision. People with more than this degree of vision loss usually use assistive technologies to access their content (and the assistive technologies usually have contrast enhancing, as well as magnification capability built into them). The 7:1 level therefore generally provides compensation for the loss in contrast sensitivity experienced by users with low vision who do not use assistive technology and provides contrast enhancement for color deficiency as well.

We exceed the AAA level of 7:1 by a factor of nearly 2.5x. For reference, our contrast of 16.10:1 is closer to black-on-white (21:1) than the WCAG requirement. I'm afraid we'll leave the colour at that.

I'm sorry – when anything except black-on-white is too low-contrast, I assume that many websites are very hard to navigate for you – for example, Wikipedia uses #202122, which has the same contrast as pretalx. Have you considered using a browser plugin to enhance text contrast on websites? There are some plugins (for every browser I looked at) that can increase text contrast, both by changing font colour and background colour. Maybe setting contrast like that could help?

**Comment by MarkNahabedian** [comment]
All of my browsers are configurted to use dark mode.  I'm not sure if the browsers on Android respect that though.

The Juliacon pretalx page renders white on black in Windows Chrome but not on Android.

Thanks for youir consideration.
